src/isar/state_machine/state_machine.py

```diff
@@ -321,8 +286,39 @@ def _check_dependencies(self):
                 return False
         return True
 
+    def publish_telemetry_thread(self) -> None:
+        robot_id = settings.ROBOT_ID
+        PUBLISH_FREQUENCY = 1
+
+        messenger = Telemetry()
+
+        while True:
+            timestamp = datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ")
+
+            state: IsarStatePayload.State = IsarStatePayload.State(self.current_state)
+            payload_state = IsarStatePayload(state=state, timestamp=timestamp).to_json()
+
+            battery_payload: int = 100
+            payload_battery = TelemetryBatteryPayload(
+                battery=battery_payload, timestamp=timestamp
+            ).to_json()
+
+            pose: Pose = self.robot.get_pose()
+            euler_orientation: np.ndarray = pose.orientation.to_euler_array()
```

Comment:
Why do we prefer to publish in euler rather than keep the quaternions?

Reply:
The reason was the draft form mqtt-api in flotialla-openapi, but I agree it makes sense to continue with quaternions